tools/fix_auth/fix_auth.rb

```diff
@@ -89,6 +93,7 @@ def run
 
       generate_password if options[:key]
       fix_database_yml if options[:databaseyml]
+      load_rails if options[:allow_failures]
```

Comment:
I don't get why you load rails only when you allow failures

Reply:
@Fryguy we have never loaded rails before. As long as the database.yml is fixed (line above) I guess we could load it.

It certainly would mask some issues but would simplify this process (typically performed in a critical window).